sos/report/plugins/crio.py

```diff
@@ -61,7 +61,7 @@ def setup(self):
             'pods',
             'ps',
             'ps -a',
-            'ps -v',
+            'ps -va',
```

Comment:
Isn't collecting `ps -a` redundant, then? Or are *both* outputs valuable to collect and use, as the output format differs a lot?

Reply:
The `ps -a` collects a "short" output for all containers (including the non-running ones), while `ps -va`  collects a verbose output (and the `a` added by this PR is to include also the non-running ones). I think both are useful.